bld/namelist_files/namelist_definition_ctsm.xml

```diff
@@ -636,6 +636,46 @@ The maximum value to use for zeta under stable conditions
 baseline proportion of nitrogen allocated for electron transport (J)
 </entry>
 
+<entry id="use_mosslichen" type="logical" category="physics"
+        group="clm_inparm" valid_values="" value=".false.">                                                    
+Toggle to turn on moss&lichen module. "use_mosslichen" can only be used together with use_fates at the moment                         
```

Comment:
Special characters like `&` should be replaced by their HTML equivalent code. In this case, `&amp;`.
For an example see line 328 in this file: https://github.com/MetOs-UiO/CTSM/blob/88b6532714ea3bfb294234148884811f3ad5572e/bld/namelist_files/namelist_definition_ctsm.xml#L328.

Reply:
Thanks for pointing this out.